### ISSUE TITLE
Accessibility, Section 508 Compliance: List of APIs widget keyboard trap

### ISSUE THREAD

**Issue by jdm4344** [Azure/api-management-developer-portal#1374]
## Bug description

After opening the tag filter popup within the List of APIs widget, a keyboard trap is created in the popup elements that cannot be exited without navigating to another page or interacting via mouse.

## Reproduction steps

Using a List of APIs widget:
1.  Move keyboard focus to the Tag Search button on the widget using Tab.
2. Open the Tag Search popup using Spacebar or Enter.
3. The user is able to tab between the search icon and search field but unable to close the popup or move tab focus out of the popup without interacting with the mouse or navigating to another page.

## Expected behavior

The Tag Search popup is able to be entered and then exited with focus returning to the next logical place in the page's tab order after the button which opened the popup using only keyboard controls

## Is your portal managed or self-hosted?

Managed

## Environment

 - Operating system: MacOS 10.15.7 - Macbook Pro 15-inch, 2017 model
 - Browser: Google Chrome
 - Version: 91
 
## Additional context

Affected widget:
<img width="658" alt="Keyboard-Trap" src="https://user-images.githubusercontent.com/31706474/125684024-5899b6c7-4a8b-4408-9415-e5a131b6e41b.png">

Related Section 508 test step: https://section508coordinators.github.io/TrustedTester/keyboard.html#check-212-no-keyboard-trap

**Comment by christopherwoodland** [comment]
Do we have a sense of when #1454 will be merged?

Thanks
